docs/low-code-spark/gems/source-target/file/delta.md

```diff
@@ -228,11 +228,13 @@ object writeDelta {
 
 ## Delta MERGE
 
-### Upsert data with Delta
+Delta merge lets us take advantage of three types of slowly changing dimension(SCD) approaches: SCD1, SCD2, and SCD3. These approaches refer to record modifications in which history is not retained (SCD1), history is retained at the row level (SCD2), or history is retained at the column level (SCD3).
```

Comment:
(Nitpick) It's not the only use case for Delta MERGE, but it's the mechanism that allows us to do SCD.